engine/common/filesystem.c

```diff
@@ -463,6 +463,22 @@ static const char *FS_FixFileCase( const char *path )
 	return path;
 }
 
+#if XASH_WIN32
+/*
+====================
+FS_PathToWideChar
+
+Converts input UTF-8 string to wide char string.
+====================
+*/
+const wchar_t *FS_PathToWideChar(const char *path)
+{
+	static wchar_t pathBuffer[MAX_PATH];
```

Comment:
Isn't MAX_PATH is cringe 260 bytes while Windows API can potentially use more? Or it can't?
MS docs says it still can't and for long path support engine must be patched and it must be enabled in registry. Oh well...

Also please follow our code style ^^

Reply:
Only Windows 10 supports long path. Ok I fixed code style.